src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./values.scss";
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  font-family: "Roboto", sans-serif;
+
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  display: flex;
+  flex-direction: column;
+  background-color: $white;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    margin-top: 0;
+  }
+
+  &__subtitle {
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+    width: 100%;
+  }
+
+  &__reviews {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__reviews-title {
+    color: $main-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    color: $secondary;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-amount {
+    color: $main-accent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    margin: 0;
+  }
+}
+
+.stars {
+  box-sizing: border-box;
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-position: center;
+
+    .stars--4 &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
+
+.button {
+  box-sizing: border-box;
+  background-color: $blue-accent;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: $white;
+  text-align: center;
+  padding: 5px;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border: 1px solid $blue-accent;
+  width: 166px;
```

Comment:
the button stretches across the entire available width of the card, so it’s better to specify the value as a percentage
because if the width of the card changes, the button must adapt